packages/server/src/seed/seed.controller.ts

```diff
@@ -168,7 +168,7 @@ export class SeedController {
     return 'Data successfully seeded';
   }
 
-  @Get('fillQueue')
+  @Get('fill_queue')
```

Comment:
What is the reason for this?

Reply:
I figured I'd reformat to use underscore so we could follow the convention we were using before
https://github.com/sandboxnu/office-hours/blob/b9457f6c5bd052b1bebf9f20ad5ed9ef265e87ed/packages/server/src/course/course.controller.ts#L62